docs/utils.md

```diff
@@ -19,9 +19,9 @@ This component requires the same CouchDB options as other components in this mod
 | `couch.userDbName` | `{String}` | The CouchDB database name containing our user records.  Defaults to `users`. |
 | `couch.userDbUrl`  | `{String}` | The URL of our CouchDB instance.  By default this is expanded from `userDbName` above using the pattern `http://admin:admin@localhost:%port/%userDbName` (see above). |
 
-## Component Invokers
+### Component Invokers
 
-## `{that}.createNewUser(userData)`
+### `{that}.createNewUser(userData)`
```

Comment:
Anything that takes an argument should have a breakdown of each param and type.  I know you did this in the next sentence, but it should really match the syntax used in the rest of the package, as shown here:

https://github.com/GPII/gpii-express-user/blob/a83beacdffb4096a379fe91a8f6e23979839fdd8/docs/forgotComponent.md#thathandlerequestresponseerror-response-body

Reply:
I think this should be matched up with the preferred format now.

Reply:
Same comment about making it clear that it's a `fluid.promise` with a link to the docs.